src/Core_Command.php

```diff
@@ -158,6 +161,16 @@ public function download( $args, $assoc_args ) {
 			$download_url = str_replace( '.zip', '.tar.gz', $offer['download'] );
 		}
 
+		if ( true === \WP_CLI\Utils\get_flag_value( $assoc_args, 'skip-content' ) ) {
+			$offer = unserialize( self::_read( 'https://api.wordpress.org/core/version-check/1.6/' ) );
+			$download_url = $offer['offers'][0]["packages"]["no_content"];
+			$version = $offer['offers'][0]["current"].' without content';
+		}
+
+		if ( true === \WP_CLI\Utils\get_flag_value( $assoc_args, 'skip-content' ) && 'en_US' !== $locale ) {
+			WP_CLI::error( 'The skip content build is only available for the en_US locale.' );
+		}
```

Comment:
I think it would make sense to render this error earlier on, to avoid an unnecessary HTTP request when the error will render.

Also, did you confirm this is the case? That `--skip-content` is available for the `en_US` locale?

Reply:
Thanks @danielbachhuber, I've moved this check to earlier on. 
Yes, as I understand, `--skip-content` is only available for the `en_US` locale.